engines/python/setup/djl_python/fastertransformer.py

```diff
@@ -0,0 +1,50 @@
+import fastertransformer as ft
+from djl_python import Input, Output
+import logging
+from typing import Optional
+
+class FasterTransformerService(object):
+    def __init__(self) -> None:
+        self.initialized = False
+
+    def initialize(self, properties):
+        self.tensor_parallel_degree = int(properties.get("tensor_parallel_degree", 1))
+        self.pipeline_parallel_degree = int(properties.get("pipeline_parallel_degree", 1))
+        self.dtype = properties.get("dtype", "fp32")
+        self.model_id = properties.get("model_id")
+        self.model = self.load_model()
+        self.initialized = True
+
+    def load_model(self):
+        logging.info(f"Loading model: {self.model_id}")
+        return ft.init_inference(self.model_id, self.tensor_parallel_degree,
+                                    self.pipeline_parallel_degree, self.dtype)
+
+    def inference(self, inputs: Input):
```

Comment:
add a parameter field to allow customer do customized setup on beam search and other settings

Reply:
https://github.com/deepjavalibrary/djl-serving/blob/master/engines/python/setup/djl_python/huggingface.py#L127

Reply:
done